src/ApplicationInsights.Kubernetes/Debuggings/LogListener.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using Microsoft.Extensions.DiagnosticAdapter;
+
+namespace Microsoft.ApplicationInsights.Kubernetes.Debugging
+{
+    /// <summary>
+    /// The default listener of Application Insights for Kubernetes.
+    /// </summary>
+    public sealed class LogListener
+    {
+        private DiagnosticLogLevel _minimumLevel = DiagnosticLogLevel.Warning;
+        private LogListener() { }
+        static LogListener() { }
+        /// <summary>
+        /// Gets the singleton instance of Application Insights for Kubernetes diagnostic source listener
+        /// </summary>
+        /// <returns></returns>
+        public static LogListener Instance { get; } = new LogListener();
```

Comment:
Why is this a singleton?

Reply:
Yeah, right, that doesn't make any sense. Updated.